### ISSUE TITLE
Import appears to hang while database is backing up

### ISSUE THREAD

**Issue by thisisjamessmith** [Pennebaker/craftcms-thearchitect#63]
Attempting to run an import on a site whose database is 1.2GB causes the UI to hang while the database backs up to the storage folder. Can we make the automated DB backup optional?

**Comment by spAnser** [comment]
Absolutely. It might be best to create a backup of the structures only instead of a full content backup.

PS is this for Craft 3?

**Comment by hisnameisjimmy** [comment]
I'm running into this same issue. We have a large database (over a dozen languages, thousands of pages, etc), and it times out during the backup stage. This is on a CPU optimized Digital Ocean box and as soon as I do a raw import I can see MySQL hit 100% right away until the action times out.

This is with 1.6.0 beta 6 on Craft CMS 2.

Super similar to: https://github.com/Pennebaker/craftcms-thearchitect/issues/38
